src/store/singing.ts

```diff
@@ -193,6 +194,15 @@ export const singingStore = createPartialStore<SingingStoreTypes>({
               x.speakerUuid === userOrderedCharacterInfos[0].metas.speakerUuid // FIXME: defaultStyleIds内にspeakerUuidがない場合がある
           )
         ].defaultStyleId;
+      */
+      /* 
+         Sing/CharacterMenuButton.vue内のgetDefaultStyle関数内の問題と同じ
+         対処療法として最初のスタイルを仮のスタイルにしている
+      */
+      const defaultStyleId =
+        userOrderedCharacterInfos[0].metas.styles[0].styleId;
+
+      const styleId = singer?.styleId ?? defaultStyleId;
```

Comment:
ここにはsing可能なキャラしか来ないので、このワークアラウンドで問題なさそうに思いました！！

細かいですが、コメントで参照先を書くと、参照先の名前などが変わったやそちらだけfixされた場合に、何を目的としていたコメントなのか追いづらくなるかもと思いました。
あと`/**/`より`//`のが行数少なくて良い感じかも。

なのでこんな感じにさせていただきます！！

```suggestion
      // 最初のスタイルをソングエディタにおける仮のデフォルトスタイルとする
      // TODO: ソングエディタ向けのデフォルトスタイルをどうするか考える
      const defaultStyleId =
        userOrderedCharacterInfos[0].metas.styles[0].styleId;

      const styleId = singer?.styleId ?? defaultStyleId;
```